frontend/appflowy_flutter/packages/flowy_infra_ui/lib/widget/error_page.dart

```diff
@@ -1,11 +1,201 @@
+import 'package:flowy_infra/image.dart';
+import 'package:flowy_infra_ui/flowy_infra_ui.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:url_launcher/url_launcher.dart';
 
 class FlowyErrorPage extends StatelessWidget {
-  final String error;
-  const FlowyErrorPage(this.error, {Key? key}) : super(key: key);
+  factory FlowyErrorPage.flowyError(
+    Error e, {
+    required String howToFix,
+    Key? key,
+  }) =>
+      FlowyErrorPage._(
+        e.toString(),
+        stackTrace: e.stackTrace?.toString(),
+        howToFix: howToFix,
+        key: key,
+      );
+
+  factory FlowyErrorPage.error(
```

Comment:
I wanted to make a constructor to get the `FlowyErrorPage` from a `FlowyError`, but sadly it isn't available for `flowy_infra_ui`.

Reply:
Do you mean we can't reference to the `FlowyError`? If yes, how about adding the deps?

Reply:
Yes, `FlowyError` is generated and available in the backend package. I'll try to add the dependency and get back to you.